job.go

```
@@ -233,20 +174,31 @@
+
 // close closes the job and its associated channels.
 // the job regardless of its current state, except when locked.
-func (j *job[T, R]) close() error {
+func (j *job[T]) close() error {
 	if err := j.isCloseable(); err != nil {
 		return err
 	}
 
-	j.resultChannel.Close()
-	j.Ack()
+	j.ack()
```

golangci-lint (1.64.8)

Error return value of `j.ack` is not checked

(errcheck)